bootstrap_vagrant.sh

```diff
@@ -1,10 +1,9 @@
 #!/bin/sh
 set -xe
 type git || yum -y install git || (apt-get update; apt-get -y install git)
-if grep -q 6.0 /etc/debian_version ; then
-  # Squeeze backports
-  echo "deb http://mirror.bytemark.co.uk/debian-backports squeeze-backports main" > /etc/apt/sources.list.d/backports.list
-  echo "Package: *\nPin: release a=squeeze-backports\nPin-Priority: 500\n" > /etc/apt/preferences.d/backports
```

Comment:
Yes, having backports is an exception and we only had that in 6.0/squeeze, which is not foreman-supported now anyway.